src/backend/routers/students.ts

```diff
@@ -17,30 +16,85 @@ export const studentProcedures = {
 
       return result;
     }),
+  //* this function gets students for the current logged in case manager
```

Comment:
I would either introduce a new procedure or rename this to make the functionality more clear, maybe something like `getMyAssignedStudents()`?

Reply:
I might suggest either getMyStudents() (there's no other context in which anyone can 'have' students, right?) or getAssignedStudents() (if the convention is to not give a perspective to the procedures). If this procedure's name is changed, should the `AllStudents.tsx` component also have its name changed?

Reply:
Ok, I changed the name for that function to "getMyStudents", to be in alignment with the "getMe" function name and the future "getMyStaff"/"getMyParas" name (Because we wouldn't say "getAssignedStaff" since I think the dynamic between Paras and Case Managers is different than that of students and Case Managers.